encoder.go

```diff
@@ -65,6 +65,8 @@ func (e *Encoder) Encode(db *Database) error {
 		if err = db.Hashes.writeTo(e.w); err != nil {
 			return err
 		}
+
+		db.Content.Meta.HeaderHash = ""
```

Comment:
We should probably add a comment here as well that describes why we can just set it to a blank value

Reply:
I'll add a corresponding comment after merge